net/interfaces/interfaces_bsd.go

```diff
@@ -2,11 +2,10 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// This might work on other BSDs, but only tested on FreeBSD.
-// Originally a fork of interfaces_darwin.go with slightly different flags.
+// Common code for FreeBSD and Darwin.
```

Comment:
Leave a comment that it might work on other BSDs too but untested? Notably OpenBSD, which we run on.